spec/integration_spec.rb

```diff
@@ -1257,26 +1263,20 @@ class SerializedDocument < ActiveRecord::Base
     expect(index.primary_key).to eq('card_number')
   end
 
-  it 'adds custom complex attribute' do
-    People.create(first_name: 'Jane', last_name: 'Doe', card_number: 75_801_887)
-    result = People.raw_search('Jane')
-    expect(result['hits'][0]['full_name']).to eq('Jane Doe')
-  end
-
   it 'does not call the API if there has been no attribute change' do
     person = People.search('Jane')[0]
-    before_save_statuses = People.index.get_all_update_status
-    before_save_status = before_save_statuses.last
+    before_save_statuses = People.index.tasks['results']
+    before_save_status = before_save_statuses.first
```

Comment:
Now, the most recent tasks are at the top of the list. In meilisearch v0.24.0 they were at the end of the list.

Reply:
not sure to get exactly what you want. Can you suggest changes? :)

Reply:
Looks good to me to cover the test case. I even remove the second part since the test was doing twice the same test.